### ISSUE TITLE
Add label "API Review Scoping" in GH issues of API Review meeting type

### ISSUE THREAD

**Issue by maririos** [Azure/azure-sdk-tools#4108]
As mentioned in https://github.com/Azure/azure-sdk-tools/issues/3858

**Comment by maririos** [comment]
@maririos do we need to add it if the issue already exists?

**Comment by maririos** [comment]
It should always exist, so we need to add it for new issues, and for existing issues make sure it is there

**Comment by maririos** [comment]
@JonathanCrd is figuring out the permissions for the Azure SDK bot into our testing repository

**Comment by JonathanCrd** [comment]
> @JonathanCrd Jonathan Cardenas FTE is figuring out the permissions for the Azure SDK bot into our testing repository

Now we have 2 new testing repos in the bot account, so we should not be blocked by permissions anymore :)

**Comment by JonathanCrd** [comment]
I have modified the current flows involved in the GH Issue creation, I added an optional parameter that expects a list of labels to be included on the issues. I also modified the `GitHub - Close - issue` flow to add a label when reopening an issue (if label is given).

This way, this is not a breaking change for the apps, they should be working as normal.

Next step:
- [ ] Adding the Label parameter in the API Reviews App (Blocked until we finished working on the Daylight Savings bug)
